pkg/kgo/ring.go

```diff
@@ -40,19 +45,19 @@ const (
 	eight = mask7 + 1
 )
 
-type ringReq struct {
+type fixedRing[T any] struct {
```

Comment:
I'm fine with fixed being deleted.

Reply:
Done in https://github.com/twmb/franz-go/pull/1057/commits/f84c863fc500d95b8680220f7d4f93e547a10f44 and https://github.com/twmb/franz-go/pull/1057/commits/5839204b04294cd718afa74cd4fc373b8d949ee8